### DISCUSSION TITLE
The script "gh-pages -d build" never ends

### DISCUSSION THREAD

**Discussion by jozinho22** [community/community#57339] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,

I have a react app i want to deploy to github pages.
I followed the tuto.

When I run "npm deploy", the script "npm run build" goes well but the script "gh-pages -d build" never ends and I have no description of what is the problem.

I add 
`"homepage": "https://jozinho22.github.io/maths",`
and this
```
 "scripts": {
    "start": "BROWSER=none react-scripts start",
    "predeploy": "npm run build",
    "deploy": "gh-pages -d build",
    "build": "CI=false react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
```

All the repo is ready because I use it for a while. 
So is it possible to have log any help from the script launch ?

Thank you

**Comment by mateobongiorno** [comment, marked as answer]
Hi there!

To get more insight into the problem, you can modify your script to include the '--verbose' flag. This will provide you with additional output and hopefully give some light on what's causing the deployment to hang. 

Here's how you can update your file:
`"deploy": "gh-pages -d build --verbose",`

It will give you a more detailed log of the deployment process, which should help in identifying the problem.

**Comment by pokpok13** [comment]
/Putanidamong animal pokpok anakputang magatal a puking pokpok a hudas

**Comment by pokpok13** [comment]
Magatal a puking pokpok a hudas
